broker/src/main/java/org/apache/rocketmq/broker/client/net/Broker2Client.java

```diff
@@ -96,7 +97,7 @@ public void notifyConsumerIdsChanged(
         try {
             this.brokerController.getRemotingServer().invokeOneway(channel, request, 10);
         } catch (Exception e) {
-            log.error("notifyConsumerIdsChanged exception, " + consumerGroup, e.getMessage());
+            log.error("notifyConsumerIdsChanged exception. group={}, error={}", consumerGroup, e.toString());
```

Comment:
why not just write as `log.error("xxxxxx", e);`?

Reply:
Because it may cause too many stacktraces. These error occurs when client is not online.